Full Store Editing: Technical documentation (#5939)
woocommerce/woocommerce-gutenberg-products-block: docs/readme.md

| [Store API (REST API)](../src/StoreApi/README.md) | These documents cover the Store API used to get product data on the frontend.                                                                                                                                                                                                   |
| [Extensibility](extensibility/README.md)          | These documents cover extensibility of WooCommerce Blocks.                                                                                                                                                                                                                      |
| [Theming](theming/README.md)                      | These documents cover theming for blocks, styles, CSS classnames and other theming best practices.                                                                                                                                                                              |

> | [Templates](templates/README.md)                  | These documents cover WooCommerce block template (and parts) functionality and technical overview and related information.                                                                                                                                                      |

Reviewer: ```suggestion
| [Templates](templates/README.md)                  | These documents provide a technical overview of WooCommerce block template (parts) functionality.                                                                                                                                                      |
```
